include/koa/authentication/passport-strategies/salesforce_authentication_strategy.js

```diff
@@ -2,20 +2,21 @@ const CustomStrategy = require('passport-custom').Strategy;
 const strategyServices = require('./strategy_services');
 const request = require('request-promise');
 const Promise = require('bluebird');
-let SALESFORCE_OAUTH_TOKEN_URL, SALESFORCE_OAUTH_AUTHORIZE_URL, state, SALESFORCE_API_URL;
+let state, salesforceAPIUrl;
 const isSandbox = process.env.USE_SALESFORCE_SANDBOX === 'true';
+const OAUTH_TOKEN_SERVICE = '/services/oauth2/token';
```

Comment:
Good Change, if these are not 
A.) Secret
B.) Environment Dependent
Then there is no reason to have them in the .evn files.